app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -80,6 +117,30 @@
   }
 }
 
+.js-enabled {
+  .gem-c-feedback__option-list {
```

Comment:
Just wondering why there's so many rules nested beneath `.js-enabled` - doesn't the component simply not appear if JS is disabled?

Reply:
Yes, good point, I don't think this is needed any more since I tidied up some of the CSS. Will remove